src/options.ts

```diff
@@ -45,14 +45,21 @@ type SpanProcessorFactory = (
 
 type PropagatorFactory = (options: Options) => TextMapPropagator;
 
+export enum TracesExporter {
+  JAEGER = 'jaeger',
+  JAEGER_THRIFT_SPLUNK = 'jaeger-thrift-splunk',
+  OTLP = 'otlp',
```

Comment:
do we want this to be `otlp-grpc`?

Reply:
It's HTTP, ingress doesn't currently support grpc.
[The spec](https://github.com/signalfx/gdi-specification/blob/aea51ebe986de6980950ca3f7948d84420960d8b/specification/configuration.md) says that the default has to be `otlp`.

Reply:
Sorry I didn't mean gpprc specifically but just to add the format instead of just calling it otlp but if the spec recommends otlp then I suppose we use otlp as an alias for otlp-http.

Reply:
I get where you are coming from, but I'd not add aliasing yet and leave only the minimal implementation per spec for now.

Reply:
See https://github.com/signalfx/splunk-otel-js/pull/218#discussion_r695641719